package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "stylelint-create",
+  "version": "0.0.1",
+  "description": "The npm init script for Stylelint.",
+  "keywords": [
+    "stylelint",
+    "npm init"
+  ],
+  "repository": "stylelint/create-stylelint",
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/stylelint"
+  },
+  "license": "MIT",
+  "author": "stylelint",
+  "main": "create-stylelint.mjs",
+  "bin": {
+    "create-stylelint": "./create-stylelint.mjs"
+  },
+  "scripts": {
+    "format": "prettier . --write --cache",
+    "lint": "npm-run-all --parallel --continue-on-error lint:*",
+    "lint:formatting": "prettier . --check --cache",
+    "lint:js": "eslint . --cache --max-warnings=0 --ext .js,.mjs",
+    "lint:md": "remark . --quiet --frail",
+    "lint:packagejson": "npmPkgJsonLint .",
+    "test": "vitest"
+  },
+  "prettier": "@stylelint/prettier-config",
+  "eslintConfig": {
+    "settings": {
+      "jest": {
+        "version": "latest"
+      }
+    },
+    "extends": [
+      "stylelint"
+    ],
+    "root": true
+  },
+  "npmpackagejsonlint": {
+    "extends": "@stylelint/npm-package-json-lint-config"
+  },
+  "remarkConfig": {
+    "plugins": [
+      "@stylelint/remark-preset"
+    ]
+  },
+  "dependencies": {
+    "execa": "^6.1.0",
+    "ora": "^6.1.2",
+    "picocolors": "^1.0.0",
+    "semver": "^7.3.8",
+    "which-pm-runs": "^1.1.0"
+  },
+  "devDependencies": {
+    "@stylelint/npm-package-json-lint-config": "^2.0.0",
+    "@stylelint/prettier-config": "^2.0.0",
+    "@stylelint/remark-preset": "^4.0.0",
+    "@types/which-pm-runs": "^1.0.0",
+    "eslint": "^8.28.0",
+    "eslint-config-stylelint": "^17.1.0",
+    "npm-run-all": "^4.1.5",
+    "remark-cli": "^11.0.0",
+    "stylelint": "^14.16.1",
+    "stylelint-config-standard": "^29.0.0",
+    "vitest": "^0.28.1"
+  },
```

Comment:
[ask] `tsconfig.json` is committed, but `typescript` is not installed into `devDependencies`. Do you think it is intentional? If so, what is the reason?

Reply:
That's probably because has https://github.com/withastro/astro/tree/main/packages/create-astro a `--typescript` flag.
see https://github.com/stylelint/stylelint-create/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R17

So Id say convert it to a simple `jsconfig.json`.
e.g.
```json
{
  "compilerOptions": {
    "checkJs": true
  }
}
```

Reply:
Ah, I was relying on typescript implicitly installed with some of the other deps, but didn't explicitly put it as a devDep in `package.json`.

Originally, the idea was to use TS for the `src/` files - see the annotations in `index.mjs`. I had thought that this setup is what most closely matches Stylelint (ex `.js` extensions, but inline TS). Any suggestions on what I should do? I can:

- formally add TS, and set up a bit more infrastructure to make it obvious/easy to use; unsure if i need to, but I can also add a compilation/build step
- remove the `tsconfig.json` and use a `jsconfig.json` as suggested
- something else!

Reply:
I understand the reason, thanks. I have no problem keeping `tsconfig.json`, while it seems good to me to install `typescript` as dev dep and add `scripts.lint:types` as `stylelint` does. Though, the latter is nice to have.

I disagree with building source files (`*.ts`) because I want to be able to run them as they are.

Reply:
Sounds good - I'll do that!